media-proxy/src/mesh/st2110.cc

```diff
@@ -125,14 +125,8 @@ void get_mtl_dev_params(mtl_init_params& st_param, const std::string& dev_port,
     st_param.log_level = log_level;
     st_param.priv = NULL;
     st_param.ptp_get_time_fn = NULL;
-    // Native af_xdp have only 62 queues available
-    if (st_param.pmd[MTL_PORT_P] == MTL_PMD_NATIVE_AF_XDP) {
-        st_param.rx_queues_cnt[MTL_PORT_P] = 62;
-        st_param.tx_queues_cnt[MTL_PORT_P] = 62;
-    } else {
-        st_param.rx_queues_cnt[MTL_PORT_P] = 128;
-        st_param.tx_queues_cnt[MTL_PORT_P] = 128;
-    }
+    st_param.rx_queues_cnt[MTL_PORT_P] = 17;
+    st_param.tx_queues_cnt[MTL_PORT_P] = 17; 
```

Comment:
Please leave a visible comment before these lines to explain why these seventeens are hardcoded here.

Reply:
Fixed in 464e73822b867e18a244efc9a09613f678f0a633